internal/pkg/cli/svc_init.go

```diff
@@ -107,6 +107,7 @@ var serviceTypeHints = map[string]string{
 	manifest.LoadBalancedWebServiceType:  "Internet to ECS on Fargate",
 	manifest.BackendServiceType:          "ECS on Fargate",
 	manifest.WorkerServiceType:           "Events to SQS to ECS on Fargate",
+	manifest.StaticSiteType:              "S3 Bucket for static assets",
```

Comment:
What do you think of this boss?
```suggestion
	manifest.StaticSiteType:              "Internet to CDN to S3 bucket",
```

Reply:
🤔 it's not only for cdn and possibly for ALB right